app/src/main/java/ch/blinkenlights/android/vanilla/PlaybackActivity.java

```diff
@@ -733,12 +735,14 @@ else if (group == CTX_MENU_GRP_FINISH)
 	 *
 	 * @param songIntent intent containing song id as {@link Long} in its "id" extra.
 	 */
+	@SuppressLint("WrongConstant") // flag is ignored on Android < 7.0
 	protected void queryPluginsForIntent(Intent songIntent) {
 		// obtain list of plugins anew - some plugins may be installed/deleted
 		mPlugins.clear();
 		mLastRequestedCtx = songIntent;
 		Intent requestPlugins = new Intent(PluginUtils.ACTION_REQUEST_PLUGIN_PARAMS);
 		requestPlugins.addFlags(Intent.FLAG_INCLUDE_STOPPED_PACKAGES);
+		requestPlugins.addFlags(0x01000000); // Intent.FLAG_RECEIVER_INCLUDE_BACKGROUND
```

Comment:
Nit: can we use the a constant here instead of the magic number?

Reply:
It's not available on APIs < 26. The only thing I can do is to declare it right there in PlaybackActivity with relevant javadoc

Reply:
But its a constant, so only the compiler would care, no?

The produced jar / apk would just contain the raw value, so older API levels would be happy.